src/data/monitoring.json

```diff
@@ -203,16 +200,17 @@
           "404": {
             "$ref": "#/components/responses/IndexNotFound"
           }
-        },
-        "tags": ["monitoring"]
+        }
       }
     },
     "/1/status": {
       "get": {
-        "operationId": "getClustersStatus",
+        "operationId": "getStatus",
```

Comment:
we also have a `getStatus` in analytics, which is why the monitoring one was renamed to `getClustersStatus`. Not sure what the best way to reconcile this would be though 🤔

Reply:
> Not sure what the best way to reconcile this would be though 🤔

Actually maybe we could try namespacing the tools names
`getStatus` -> `monitoring:getStatus` / `analytics:getStatus`

Reply:
that would make sense indeed!